specification/loadtestservice/resource-manager/Microsoft.LoadTestService/preview/2022-04-15-preview/loadtestservice.json

```diff
@@ -0,0 +1,533 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LoadTestResource"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string"
+        }
+      }
+    },
+    "LoadTestResource": {
+      "type": "object",
+      "description": "LoadTest details",
+      "properties": {
+        "properties": {
+          "description": "Load Test resource properties",
+          "$ref": "#/definitions/LoadTestProperties",
+          "x-ms-client-flatten": true
+        },
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity",
+          "description": "The type of identity used for the resource."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "LoadTestResourcePatchRequestBody": {
+      "description": "LoadTest resource patch request body.",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object"
+        },
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity",
+          "description": "The type of identity used for the resource."
+        },
+        "properties": {
+          "description": "Load Test resource properties",
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "properties": {
+            "description": {
+              "description": "Description of the resource.",
+              "maxLength": 512,
+              "type": "string"
+            },
+            "encryption": {
+              "description": "CMK Encryption property.",
+              "type": "object",
+              "$ref": "#/definitions/EncryptionProperties"
+            }
+          }
+        }
+      }
+    },
+    "LoadTestProperties": {
+      "description": "LoadTest resource properties.",
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "Description of the resource.",
+          "maxLength": 512,
+          "type": "string"
+        },
+        "provisioningState": {
+          "description": "Resource provisioning state.",
+          "$ref": "#/definitions/ResourceState",
+          "readOnly": true
+        },
+        "dataPlaneURI": {
+          "description": "Resource data plane URI.",
+          "maxLength": 2083,
+          "type": "string",
+          "readOnly": true
+        },
+        "encryption": {
+          "description": "CMK Encryption property.",
+          "type": "object",
+          "$ref": "#/definitions/EncryptionProperties"
+        }
+      }
+    },
+    "EncryptionProperties": {
```

Comment:
>EncryptionProperties [](http://example.com/codeflow?start=5&length=20)

can you please include this property in your patch example?

Reply:
@jorgecotillo i have updated the examples to contain encryption properties